test/integration/acceptance/custom/basic/basic.go

```diff
@@ -68,13 +69,41 @@ func (r *BasicTestRunner) Setup(ctx context.Context, createOptsPublic types.Site
 	err = prv1Cluster.VanClient.RouterCreate(ctx, *siteConfig)
 	assert.Assert(t, err)
 
+	var podStartTimeBefore *v1.Time
+	if testSync == true {
+		// Pick the pod details for config-sync validation
+		pods, _ := kube.GetPods("skupper.io/component=router", prv1Cluster.Namespace, prv1Cluster.VanClient.KubeClient)
```

Comment:
I just wanted to mention that the pod contains two containers (`router` and `config-sync`) associated to the same label, `router`. This is an example:
```
Labels:       app.kubernetes.io/name=skupper-router
              app.kubernetes.io/part-of=skupper
              application=skupper-router
              pod-template-hash=6d9c9d7769
              **skupper.io/component=router**
Annotations:  prometheus.io/port: 9090
              prometheus.io/scrape: true
Status:       Running
IP:           172.17.0.3
IPs:
  IP:           172.17.0.3
Controlled By:  ReplicaSet/skupper-router-6d9c9d7769
Containers:
  router:
    Container ID:   docker://15f0a41ac43165a777a85e0f7701cac259e63b318402a85bf0c0eae4432a4268
    Image:          quay.io/skupper/skupper-router:main
    Image ID:       docker-pullable://quay.io/skupper/skupper-router@sha256:99dd1f23e8b2f99fbd28fa972b70cef710c49dfa319450d9df84c7ae0efd8bad
    Ports:          5671/TCP, 9090/TCP, 55671/TCP, 45671/TCP
    Host Ports:     0/TCP, 0/TCP, 0/TCP, 0/TCP
    State:          Running
      Started:      Tue, 26 Apr 2022 13:27:45 +0200
    Ready:          True
    Restart Count:  0
    Liveness:       http-get http://:9090/healthz delay=60s timeout=1s period=10s #success=1 #failure=3
    Environment:
      APPLICATION_NAME:               skupper-router
      POD_NAMESPACE:                  default (v1:metadata.namespace)
      POD_IP:                          (v1:status.podIP)
      QDROUTERD_AUTO_MESH_DISCOVERY:  QUERY
      QDROUTERD_CONF:                 /etc/skupper-router/config/skrouterd.json
      QDROUTERD_CONF_TYPE:            json
      SKUPPER_SITE_ID:                99283926-c21f-4592-9d8f-775944023675
    Mounts:
      /etc/skupper-router-certs from skupper-router-certs (rw)
      /etc/skupper-router-certs/skupper-amqps/ from skupper-local-server (rw)
      /etc/skupper-router-certs/skupper-internal/ from skupper-site-server (rw)
      /etc/skupper-router/config/ from router-config (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-46c9m (ro)
  **config-sync:**
    Container ID:   docker://488fb4984dc37944efa6908d256140da90662e2a9041c0b6efd3e8d28501cde0
    Image:          quay.io/skupper/config-sync:latest
    Image ID:       docker-pullable://quay.io/nluaces/config-sync@sha256:039d9d991797aae02ae2e7fe5277a39abcc3b43a465f344d0d61a01d0e6ce881
    Port:           <none>
    Host Port:      <none>
    State:          Running
      Started:      Tue, 26 Apr 2022 13:27:56 +0200
    Ready:          True
    Restart Count:  0
    Environment:    <none>
    Mounts:
      /etc/skupper-router-certs from skupper-router-certs (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-46c9m (ro)
```
So, in order to filter the config-sync container (that is acting like a sidecar for the router container) you can filter by container name, in this case "config-sync". 

If you can filter that way, yo can see the paths that the containers have mounted and check if the two of them have a shared one.

Reply:
I am now inspecting both container for volume mounts